### PR TITLE
Ftu signup clean

### PR DESCRIPTION
**Pull request by nicola**

### CODE REVIEW

File: bin/ldnode.js [hunk 1]
```diff
@@ -97,6 +97,11 @@ var argv = require('nomnom')
     full: 'default-app',
     help: 'URI to use as a default app for resources (default: https://linkeddata.github.io/warp/#/list/)'
   })
+  .option('signup', {
```
Comment (dmitrizagidulin) [inline bin/ldnode.js]: Can we rename this option to `singleUserMode` (from our in-person discussion / #201?)

Comment (nicola) [inline bin/ldnode.js]: This is actually just to sign up the first user.
Single user mode right now is done by not setting the flag --idp

This is like --create-admin



On Fri, Jan 22, 2016 at 6:00 AM -0800, "Dmitri Zagidulin" <notifications@github.com> wrote:












In bin/ldnode.js:
> @@ -97,6 +97,11 @@ var argv = require('nomnom')
>      full: 'default-app',
>      help: 'URI to use as a default app for resources (default: https://linkeddata.github.io/warp/#/list/)'
>    })
> +  .option('signup', {



Can we rename this option to singleUserMode (from our in-person discussion / #201?)

Comment (dmitrizagidulin) [inline bin/ldnode.js]: Makes sense. (So this is more for https://github.com/solid/solid/issues/7). 
Can we rename it to `--create-admin` then?

Comment (nicola) [inline bin/ldnode.js]: done

**Pull request merged**